### DISCUSSION TITLE
Collect task no possible

### DISCUSSION THREAD

**Discussion by TOH2F** [glpi-project/glpi-agent#444] [Q&A]
### Bug reporting acknowledgment

Not applicable

### Professional support

None

### Describe the bug

I created a task for the collection of an element from the registry, the module is activated in GLPI agent, the agent is in 1.5, the task does not start and the log tells me

[Mon Jul 17 10:12:17 2023][info] Collect task not supported by server0
the task remains to be done in GLPI
Do you have an idea ?

GLPI 10.0.5 
Glpi inventory agent 1.5

Thanks

### To reproduce

Create task 
Add agent for collect
Add registry path
Force task

### Expected behavior

none

### Operating system

Windows

### GLPI Agent version

Other (See additional context below)

### GLPI version

10.0.5

### GLPIInventory plugin or FusionInventory for GLPI plugin version

GLPI Inventory v1.1.0

**Comment by g-bougard** [comment]
Hi @TOH2F 

this log message can tell you:
 1. you're using the wrong url for glpi server as `server` configuration for your agent, check this doc: https://glpi-agent.readthedocs.io/en/latest/configuration.html#server
 2. or such task type has been disabled on glpi server side at a global level or at the agent level, check the "Glpi Inventory" configuration in GLPI Administration

Also, you definitively should upgrade your GLPI Server & your GlpiInventory plugin to eliminate any still fixed bug issue.

**Comment by TOH2F** [comment]
Bonjour,
Thank you for response.

The module is activate globally and the agent
![image](https://github.com/glpi-project/glpi-agent/assets/60504041/f3d27d66-0c8c-41ea-93c9-ad37ba37edba)
![image](https://github.com/glpi-project/glpi-agent/assets/60504041/9e2b25c3-0f85-4fd6-9a06-d3890982e0b1)

I changed the url but nothing helps in the article (marketplace/glpiiinventory) and restart the agent

Ideas ?

![image](https://github.com/glpi-project/glpi-agent/assets/60504041/6c60854b-bbcb-4317-95c2-d757dfda50e3)

**Reply by g-bougard** [reply]
Have you forced the run of tasks ?
You should know the agent will contact the server only if it runs all tasks. By default, it contacts one time by day, but you can force that run to check the collect task is working.
